yt_dlp/extractor/vrt.py

```diff
@@ -135,11 +144,64 @@ class VRTIE(VRTBaseIE):
         },
         'params': {'skip_download': 'm3u8'},
     }]
-    _CLIENT_MAP = {
-        'vrt.be/vrtnws': 'vrtnieuws',
-        'sporza.be': 'sporza',
+    _NETRC_MACHINE = 'vrtnu'
+    _APIKEY = '3_0Z2HujMtiWq_pkAjgnS2Md2E11a1AwZjYiBETtwNE-EoEHDINgtnvcAOpNgmrVGy'
+    _CONTEXT_ID = 'R3595707040'
+    _REST_API_BASE_TOKEN = 'https://media-services-public.vrt.be/vualto-video-aggregator-web/rest/external/v2'
+    _REST_API_BASE_VIDEO = 'https://media-services-public.vrt.be/media-aggregator/v2'
+    _HLS_ENTRY_PROTOCOLS_MAP = {
+        'HLS': 'm3u8_native',
+        'HLS_AES': 'm3u8_native',
     }
 
+    _authenticated = False
+
+    def _perform_login(self, username, password):
+        auth_info = self._gigya_login({
```

Comment:
Looking at the site, I don't think this uses gigya anymore, but the vrt.be login. Is that true? I don't have an account.
If the site also uses vrt.be, can the `_perform_login` function be moved from `VrtNUIE` to `VRTBaseIE`.